azure-pipelines.yml

```diff
@@ -0,0 +1,63 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+      - develop
+
+pr:
+  autoCancel: true 
+  branches:
+    include:
+      - main
+      - develop
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+- task: UseDotNet@2
+  inputs:
+    packageType: 'sdk'
+    version: '5.0.x'
+  displayName: 'Install DotNet'
+
+- task: DotNetCoreCLI@2
+  inputs:
+    command: 'build'
+    projects: '**/**/*.csproj'
+    arguments: '--configuration $(buildConfiguration)'
+  displayName: 'DotNet Build'
+  
+
+- task: DotNetCoreCLI@2
+  inputs:
+    command: 'publish'
+    publishWebProjects: false
+    projects: '**/**/*.csproj'    
+    arguments: '-o $(Build.ArtifactStagingDirectory)/Output'
+    zipAfterPublish: False
+  displayName: 'DotNet Publish'
+
+- task: DotNetCoreCLI@2
+  inputs:
+    command: 'pack'
+    packagesToPack: '**/*.csproj'
+    versioningScheme: 'off'
+  displayName: 'DotNet Pack'
+
+- task: PublishBuildArtifacts@1
+  condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
+  inputs:
+    PathtoPublish: '$(Build.ArtifactStagingDirectory)'
+    ArtifactName: 'drop'
+    publishLocation: 'Container'
+  displayName: 'DotNet Drop'
+
+
+
```

Comment:
Blank lines

Reply:
it's done
I have deleted that line blank